app/views/shipping/shipping.scala.html

```diff
@@ -0,0 +1,57 @@
+@(subscription: model.Subscriptions.SubscriptionProduct)
+
+@import utils.Prices._
+
+@main(s"${subscription.capitalizedTitle} | Choose a Package | Subscriptions | The Guardian", isInternational=true) {
+
+    <main class="page-container gs-container">
+
+        @fragments.page.header(subscription.title, Some(subscription.description), Nil)
+
+
+        <section class="section-slice">
+            <h2 class="page-heading">@subscription.stepsHeading</h2>
+            <ol class="steps">
+                @for(step <- subscription.steps) {
+                    <li class="steps__item">@step</li>
+                }
+            </ol>
+        </section>
+
+        <section class="section-slice">
+
+            @if(subscription.insideM25) {
```

Comment:
I guess what you are trying to do is to reuse the markup while parameterising the different values. You could do that by defining a [function](https://www.playframework.com/documentation/2.2.x/ScalaTemplates#Declaring-reusable-blocks) within your view

```scala
@displayProduct(sp: SubscriptionProduct) = {
  val x = if (sp.insideM25) "a" else "b"
  <div>
    <span x="$x">
  </div>

} 
```
However, looks a bit overkill given that we repeat the same block on once and within the same file.